youtube_transcript_api/formatters.py

```diff
@@ -95,8 +95,9 @@ def _seconds_to_timestamp(self, time):
         '00:00:06.930'
         """
         time = float(time)
-        hours, mins, secs = (
-            int(time) // 3600,
+        hours= int(time) // 3600
+        time= time - hours*3600
+        mins, secs = (
             int(time) // 60,
             int(time) % 60,
         )
```

Comment:
Hi @esha71, thanks for the PR! However, I think this would actually make things more clear:
```python
hours, remainder = divmod(time, 3600)
mins, secss = divmod(remainder, 60)
```

Reply:
This looks Great! Thanks!!!